Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstandingCustomMesh.cs

```diff
@@ -33,6 +33,11 @@ private float MaxFrameTimeInSeconds
             get { return (MaxFrameTime / 1000); }
         }
 
+        /// <summary>
+        ///  Whether to create mesh colliders. If unchecked, mesh colliders will be empty and disabled.
+        /// </summary>
+        public bool CreateMeshColliders = true;
```

Comment:
There's one problem with that: setting to true would have to enable the collision meshes (potentially generating them for the first time) and this is too slow to do inside of a setter. Kicking off an async task of some sort also violates the principle of least surprise, I think. 

Let me know whether we should pursue this suggestion. I'm definitely open to making it work if there's a simple way of doing it that's consistent with the rest of the API.